pkg-manager/plugin-commands-installation/src/installDeps.ts

```diff
@@ -319,7 +320,7 @@ when running add/update with the --workspace option')
     if (opts.save !== false) {
       await Promise.all([
         writeProjectManifest(updatedProject.manifest),
-        updatedCatalogs && addCatalogs(opts.workspaceDir ?? opts.dir, updatedCatalogs),
+        (updatedCatalogs ?? opts.cleanupUnusedCatalogs) && updateWorkspaceManifest(opts.workspaceDir ?? opts.dir, { updatedCatalogs, cleanupUnusedCatalogs: opts.cleanupUnusedCatalogs }),
```

Comment:
I don't think there a need to check the existence of those settings. If they are not defined, updateWorkspaceManifest will just not do anything.

```suggestion
        updateWorkspaceManifest(opts.workspaceDir ?? opts.dir, { updatedCatalogs, cleanupUnusedCatalogs: opts.cleanupUnusedCatalogs }),
```

Reply:
Agree with this suggestion.